fcrepo4/fcrepo4
# Unsupported external-body access-types now result in 415 Unsupported …

…Media Type.

Resolves: https://jira.duraspace.org/browse/FCREPO-2586

**JIRA Ticket**: (link)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
It ensures that unsupported external-body access-types now result in 415 Unsupported Media Type.

# How should this be tested?
0. build and fire up fedora.
1. run `curl -XPUT -H"Content-Type: message/external-body; access-type=ftp; NAME=\"/some/file\"; site=\"example.com\"" -i localhost:8080/rest/external`
2. Verify that 415 http is returned.


# Additional Notes:
If a fedora binary has an invalid external-body content type associated with it,  a GET request for that resource will fail with a 415.  Previously GET requests would not fail.  

# Interested parties
@escowles
